code/modules/reagents/chemistry/recipes/pyrotechnics.dm

```diff
@@ -116,6 +116,23 @@
 	empulse(location, round(created_volume / 12), round(created_volume / 7), 1)
 	holder.clear_reagents()
 
+
+/datum/chemical_reaction/beesplosion
+	name = "Bee Explosion"
+	id = "beesplosion"
+	required_reagents = list("honey" = 1, "strange_reagent" = 1, "radium" = 1)
+	
+/datum/chemical_reaction/beesplosion/on_reaction(datum/reagents/holder, created_volume)
+	var/location = get_turf(holder.my_atom)
+	if(created_volume < 10)
+		playsound(location,'sound/effects/sparks1.ogg', 100, 1)
+	else
+		playsound(location,'sound/creatures/bee.ogg', 100, 1)
+		var/bee = created_volume/10
+		for(var/counter = 0; counter < bee; counter++)
```

Comment:
`for(var/i in 1 to bee)`

Reply:
bee isnt sanitized will it work if bee is 3.14?

Reply:
why don't you `round(created_volume/10)`?